src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -24,5 +32,8 @@ public InterceptorConfig(LoggingInterceptor loggingInterceptor) {
     @Override
     public void addInterceptors(@NonNull InterceptorRegistry registry) {
         registry.addInterceptor(loggingInterceptor);
+        registry.addInterceptor(new TokenPermissionsInterceptor())
+                .addPathPatterns(TRANSACTIONS, FILINGS);
+        registry.addInterceptor(customUserAuthenticationInterceptor);
```

Comment:
The tokenPermissionsInterceptor and customUserAuthentication should run against the same endpoints. They should run against endpoints that are called by a user, so both would need TRANSACTIONS path.
FILINGS is an api to api call so shouldn't have the user authentication done, it will need the InternalUserInterceptor at some point.

Reply:
Not sure I understand this.